pylds/vector_fields.py

```diff
@@ -181,5 +181,99 @@ def forcing(t, u, perturbation_params = [1, 0.15, 0.5]):
     
     return perturbation
 
+def HenonHeiles_vector_field(t, u, PARAMETERS = None):
+    """
+    Returns 2D Henon-Heiles vector field at time t, for an array of points in phase space.
+    Number of model parameters: 0 . PARAMETERS = [wx, wy, delta]
+    Functional form: v = (x wx^2 + 2xy, y wy^2 + x^2 - delta y^2), with u = (x, y)
+    
+    Parameters
+    ----------
+    t : float
+        fixed time-point of vector field, for all points in phase space.
+        
+    u : array_like, shape(n,)
+        points in phase space to determine vector field at time t.
+        
+    PARAMETERS : list of floats
+        vector field parameters
+    
+    Returns
+    -------
+    v : array_like, shape(n,)
+        vector field corresponding to points u, in phase space at time t
+    """
+    N_dof = u.shape[-1]
+    points_positions = u.T[:int(N_dof/2)]
+    points_momenta = u.T[int(N_dof/2):]
+    x, y = points_positions
```

Comment:
In 1DoF systems we use y for momenta, so we should avoid using it for positions elsewhere